woof-code/rootfs-petbuilds/pupmoon-font/petbuild

```diff
@@ -5,5 +5,7 @@ download() {
 build() {
     tar xJf pupmoon-font-1.0.tar.xz
     cd pupmoon-font-1.0
-    cp -arf usr/ /
+    mkdir -p etc/pupmoon
+    mv usr/share/doc/pupmoon/* etc/pupmoon
```

Comment:
Why not keep all documentation of all petbuilds, to prevent this issue from returning in other packages?

Reply:
Your call on that :)

Reply:
Can also be a case by case thing (per petbuild) if all docs are kept.

Reply:
I think it's OK to remove the deletion of /usr/share/doc from petbuilds.sh.

I can add explicit `rm -f` of large files later, where needed.

Reply:
Fair enough. I'll close this and we'll sort out deletion of docs on 'as needed' basis in each petbuild.

Reply:
Maybe the problem is that most woof-CE builds come without docx, and docx is not auto-loaded (so the result is the same as having no documentation).

How about moving the license to to a file called LICENSE or COPYING? If we do that, we can change petbuilds.so to keep files with these special names in the main package instead of moving them to _DOC.